### PR TITLE
Frontend: enable eslint no-undef and manual fix

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

File: frontends/web/src/components/anchor/anchor.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { h, RenderableProps } from 'preact';
+import { h, JSX, RenderableProps } from 'preact';
```
Comment (thisconnect) [inline frontends/web/src/components/anchor/anchor.tsx:17]: TypeScript complained about returning JSX.Element, here https://github.com/digitalbitbox/bitbox-wallet-app/pull/1027/files#diff-c46c03dfb317b41884c6e334884a9bceR26

Ideally we wouldn't have to import JSX

File: frontends/web/src/utils/env.js [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+ /* eslint no-undef: "off" */
 export const debug = process.env.NODE_ENV === 'development';
```
Comment (thisconnect) [inline frontends/web/src/utils/env.js:18]: An alternative would be to set `node: true`, here https://github.com/digitalbitbox/bitbox-wallet-app/blob/4457dde3491a63ac184e00da9f0d328d9c2fe227/frontends/web/.eslintrc.js#L3

but node is only used during build, so I prefer this way.

File: frontends/web/src/components/anchor/anchor.tsx (hunk 1 above)
Comment (benma) [reply frontends/web/src/components/anchor/anchor.tsx:17]: Why did it build successfully without importing it before?

**Review by benma — APPROVED** [review]

File: frontends/web/src/components/anchor/anchor.tsx (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/components/anchor/anchor.tsx:17]: Correction: Typescript doesn't complain but typescript-eslint. Finally found one having same issue: https://github.com/typescript-eslint/typescript-eslint/issues/2477#issuecomment-686845188

I tried going down that rabit hole (using @types/react), but stopped now and postpone until we move to TS4 + PreactX.

**Pull request merged**